File: day44/a2.py
from dataclasses import dataclass

@dataclass
class User:
    name:str
    active:bool

users =[
    User("user1",True),
    User("user2",False),
    User("user3",True)
]

user_name = []

for user in users:
    if user.active:
        user_name.append(user.name)


print(user_name)

File: day44/a2.java
import java.util.ArrayList;
import java.util.List;


record User(String name,boolean active){
    public static void main(String[] args){


        List<User> users = new ArrayList<>();
        users.add(new User("user1",true));
        users.add(new User("user2",false));
        users.add(new User("user3",true));

        List<String> userNames = new ArrayList<>();

        for(User user:users){
            if(user.active()){
                userNames.add(user.name);
            }
        }

        System.out.println(userNames);


    }
}
